templates/example_cell_clustering.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I don't think there needs to be a weighted calculation for both cluster and hCluster_cap. I think it's fine to have the default be hCluster_cap and if the users want, they can change cluster_col to cluster. Also, the output of create_c2pc_data for hCluster_cap was already created and saved to file during train_cell_som.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>